src/components/ContainerHeader.vue

```diff
@@ -29,17 +29,10 @@
                         <i class="fa fa-info" aria-hidden="true"/>
                     </a>
                 </div>
-                <div
-                    v-if="sidebarState.sidebarSection === 'user'"
-                    class="kiwi-header-option kiwi-header-option-user kiwi-header-option--active"
-                >
-                    <a @click="sidebarState.close()">
-                        <i class="fa fa-user" aria-hidden="true"/>
-                    </a>
-                </div>
                 <div
                     :class="{
-                        'kiwi-header-option--active': sidebarState.sidebarSection === 'nicklist'
+                        'kiwi-header-option--active': sidebarState.sidebarSection === 'nicklist',
+                        'kiwi-header-option--active': sidebarState.sidebarSection === 'user'
```

Comment:
You can't have multiple same keys in an object so this should be something like `'kiwi-header-option--active': sidebarState.sidebarSection === 'nicklist' || sidebarState.sidebarSection === 'user'`